MeetUpApp.Api/Middleware/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using MeetUpApp.Data;
+using MeetUpApp.Managers;
+
+namespace MeetUpApp.Api.Middleware.Extensions
+{
+    public static class ApplicationBuilderExtensions
+    {
+        public static IApplicationBuilder TryAddFirstUser(this IApplicationBuilder builder)
+        {
+            using var services = builder
+                .ApplicationServices.CreateScope();
+
+            var db = services.ServiceProvider
+                .GetRequiredService<AppDataContext>();
+
+            if (!db.User.Any())
+            {
+                var userManager = services
+                    .ServiceProvider.GetRequiredService<UserManager>();
+
+                userManager.AddUser("admin", "Qs3PGVAyyhUXtkRw").Wait();
+            }
+
+            return builder;
+        }
+
+        public static IApplicationBuilder AddSwaggerWithUI(
```

Comment:
Please, name it with 'Use' instead of 'Add'

Reply:
done